llvm/CheerpCmakeConf.cmake

```diff
@@ -3,7 +3,7 @@
 
 set(CMAKE_INSTALL_PREFIX /opt/cheerp CACHE STRING "")
 set(LLVM_DEFAULT_TARGET_TRIPLE cheerp-leaningtech-webbrowser-wasm CACHE STRING "")
-set(LLVM_TARGETS_TO_BUILD X86;CheerpBackend CACHE STRING "")
+set(LLVM_TARGETS_TO_BUILD X86;WebAssembly;CheerpBackend CACHE STRING "")
```

Comment:
I would like to get rid of CheerpBackend completely in this PR.
This includes removing it here, but also the directory and making it so the WebAssembly target is invoked when compiling with `-target cheerp-*`